assets/src/scripts/selectors/flood-data-selector.js

```diff
@@ -1,12 +1,17 @@
 import { createSelector } from 'reselect';
-
+import { getCurrentParmCd } from './time-series-selector';
 
 export const getFloodStages = state => state.floodData.stages || [];
 
 export const getFloodExtent = state => state.floodData.extent || {};
 
 export const getFloodGageHeight = state => state.floodState.gageHeight;
 
+export const getWaterwatchActionStage = state => state.floodState.actionStage;
```

Comment:
If we put the stages into a single floodLevel property, I don't think these individual stage ones are necessary

Reply:
Placed all data in to a single flood property.